asserts/store_asserts.go

```diff
@@ -48,6 +49,11 @@ func (store *Store) URL() *url.URL {
 	return store.url
 }
 
+// FriendlyStores returns stores to be treated as extensions of this store.
```

Comment:
Isn't that reversed?  We don't want a store assertion listing every single extension of the parent store. We want child stores to mention their parent.